README.rst

```diff
@@ -11,88 +11,88 @@ more information about the backend service.
 
 aiven-client (``avn``) is the official command-line client for Aiven.
 
-::
-
-                        `'+;`         `'+;`
-    The Aiven Crab    '@@@#@@@`     '@@@#@@@`
-    ~~~~~~~~~~~~~~   #@.     #@.   @@.     #@.
-                     @: ,@@   @@   @: ,@@   @@
-                    ,@  @@@@@ :@  :@  @@@@@ .@
-                     @  #@@@. #@   @` #@@@` @@
-                     @@      `@#   @@      `@#
-                      @@#. :@@+     @@#. :@@#
-                       `+@@@'        `#@@@'
-               ,;:`                             ,;;.
-             @@@@@@#     .+@@@@@@@@@@@@@'.    `@@@@@@@
-            @@@@@#    @@@@@@@@@@@@@@@@@@@@@@+    @@@@@@
-             @@@   ;@@@@@@@@@@@@@@@@@@@@@@@@@@@`  `@@;
-              `  `@@@@@@@@@@@        ;@@@@@@@@@@@
-          `@@@  '@@@@@@@@@@@@@       @@@@@@@@@@@@@`  @@@
-         '@@@` .@@@@@@@@@@@@@@@    `@@@@@@@@@@@@@@@  @@@@`
-         @@@@  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  @@@@
-        '@@@@  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  @@@@
-        ,:::;  @@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  ,:::
-           :@  ,@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@@  #@
-           @@@  +@#+#@@@@@@@@@@@@@@@@@@@@@@@@@#+#@.  @@@
-           @@@@        '@@@@@@@@@@@@@@@@@@@.        @@@@
-           @@@  @@@@@@+  @@@@@@@@@@@@@@@@@  @@@@@@;  @@@
-           @@  @@@@@@@@@  @@@@@@@@@@@@@@@ `@@@@@@@@@  @+
-              @@@@@@@@@@@ :@@@@@@@@@@@@@  @@@@@@@@@@@ '
-             `@@@@@@@@@@@       ```      ,@@@@@@@@@@@
-             `@@@@@@   '@                :@:   @@@@@@
-              @@@@@:                           @@@@@@
-               @@@@@                           @@@@@
-                @@@@#                         @@@@'
-
```

Comment:
I agree that this crab doesn't add readability. But I'll miss him. How about inserting a PNG/SVG image?

Reply:
He can't stay here but I'd like to look at how to add `avn crab` as a sort of easter egg to still draw him when we want to see him